README.md

```diff
@@ -65,6 +65,7 @@ Clisso has the following commands:
     get         Get temporary credentials for an app
     help        Help about any command
     providers   Manage providers
+    status      show currently valid profiles
```

Comment:
Please keep capitalization consistent :-)
Also, it's not clear to me as a user what "valid" means, so I would maybe rephrase as follows:

`Show active (non-expired) credentials`.